.mailmap

```diff
@@ -360,6 +361,7 @@ Saúl Ibarra Corretgé <s@saghul.net> <saghul@gmail.com>
 Scott Blomquist <github@scott.blomqui.st> <sblom@microsoft.com>
 Segu Riluvan <rilwan22@hotmail.com> <riluvan@gmail.com>
 Sergey Kryzhanovsky <skryzhanovsky@gmail.com> <another@dhcp199-223-red.yandex.net>
+Sergey Zelenov <mail@zelenov.su> <sergey.zelenov@getyourguide.com>
```

Comment:
> Please keep only @zelenov.su, thanks!

Thanks.

Just in case it's not clear what's going on here: This `.mailmap` file entry results in `git` and other tools treating all instances of the other email address as the same as the `@zelenov.su` address, which is why the other address is removed from the `AUTHORS` file in this pull request. The script that generates the `AUTHORS` file refers to `.mailmap` to figure out what email address to use for people that have more than one email in the git history.

So I think we're all set.